fireapp/lib/presentation/volunteer_home/volunteer_home_widget.dart

```diff
@@ -1,8 +1,11 @@
 import 'package:fireapp/presentation/volunteer_home/volunteer_home_view_model.dart';
 import 'package:flutter/material.dart';
 import 'package:get_it/get_it.dart';
+import 'package:intl/intl.dart';
 
 import '../../base/widget.dart';
+import '../../domain/models/shift.dart';
+import '../../domain/request_state.dart';
 import '../fireapp_page.dart';
 
 class VolunteerHome extends StatelessWidget {
```

Comment:
Please combine VolunteerHome and HomeView into one StatefulWidget called VolunteerHomePage